docs/predictive-analytics/model-retraining/process.sql

```diff
@@ -0,0 +1,66 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of predictions
+CREATE STREAM PREDICTED_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Height" DOUBLE,
+  "Length" DOUBLE,
+  "Prediction" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.weight-prediction', 
+  VALUE_FORMAT = 'JSON',
+  PARTITIONS = 6
+);
+
+-- Create stream of actual weights
+CREATE STREAM ACTUAL_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Weight" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.machine-weight', 
+  VALUE_FORMAT = 'JSON',
+  PARTITIONS = 6
+);
+
+-- Create stream joining predictions with actual weights
+CREATE STREAM DIFF_WEIGHT
+WITH(
+  KAFKA_TOPIC = 'diff_weight', 
+  VALUE_FORMAT = 'JSON'
+)
+AS SELECT
+   'Key' AS "Key",
+   PREDICTED_WEIGHT."Fish_Id" AS "Fish_Id",
+   PREDICTED_WEIGHT."Species" AS "Species",
+   PREDICTED_WEIGHT."Length" AS "Length",
+   PREDICTED_WEIGHT."Height" AS "Height",
+   PREDICTED_WEIGHT."Prediction" AS "PredictedWeight",
+   ACTUAL_WEIGHT."Weight" AS "ActualWeight",
+   ROUND(ABS(PREDICTED_WEIGHT."Prediction" - ACTUAL_WEIGHT."Weight") / ACTUAL_WEIGHT."Weight", 3) AS "Error"
+FROM PREDICTED_WEIGHT
+INNER JOIN ACTUAL_WEIGHT
+WITHIN 1 MINUTE
+ON PREDICTED_WEIGHT."Fish_Id" = ACTUAL_WEIGHT."Fish_Id";
+
+-- Create table of one minute aggregates with over 15% error rate
+set 'ksql.suppress.enabled'='true';
```

Comment:
Do we allow to enable suppress in CCloud? IIRC, we don't?

Reply:
It didn't allow me to set it using the properties widget in the UI, but if it did seem to allow it in the query, and if I explain the query, it does show `ksql.suppress.enabled: true`.

Reply:
Might be a bug in CCloud... Let me ask around...

Reply:
Ok. Just got confirmation that it is a bug: https://github.com/confluentinc/ksql/pull/7716 

We should not put it into the recipe. Hope this does not become a blocker for it? (Btw: it's on our Q1 roadmap to work on `EMIT FINAL` as we want to enable it cloud -- so maybe(!) it could become available in Q2...)

Reply:
@daveklein @mjsax so this line should be removed from the recipe and re-validated?

Reply:
I removed `EMIT FINAL` and `ksql.suppress.enabled=true` and re-tested the application.  It seems to work fine.